includes/class-wc-payments-order-success-page.php

```diff
@@ -84,6 +84,11 @@ public function show_woocommerce_payments_payment_method_name( $payment_method_t
 			return $this->show_card_payment_method_name( $order, $payment_method );
 		}
 
+		if ( $payment_method->get_id() === Payment_Method::GRABPAY ) {
+			$output = $this->show_payment_method_logo( $gateway, $payment_method );
+			return false !== $output ? $output : $payment_method_title;
+		}
```

Comment:
I see that the `show_payment_method_logo()` method was added, with an optional `$show_payment_method_name` argument - `false` by default.

But since `show_payment_method_logo()` is only used here, and since `$show_payment_method_name` will be `false` - what is the difference with the `show_lpm_payment_method_name()` method? Can we just use `show_lpm_payment_method_name()`?

Reply:
You are right, we can re-use the existing function, I have updated the tests as well and included the `title` attribute so it displays the payment method name on hover.

Reply:
Nice, thank you! I like that this addition will also improve existing LPMs!